qa/tasks/barbican.py

```diff
@@ -0,0 +1,517 @@
+"""
```

Comment:
@shyukri could you drop the change to this file? as mimic does not have `qa/tasks/barbican.py` .